.github/ISSUE_TEMPLATE/new-repository.yaml

```diff
@@ -0,0 +1,48 @@
+name: New Repository
+description: Request the addition of a new repository.
+body:
+  - type: input
+    id: repository-url
+    attributes:
+      label: Repository URL
```

Comment:
should this just be name rather than url?

Reply:
It could be! I liked url as there are lots of repos with similar names, so it just makes it super clear